### PR TITLE
Add Happi package.

### PR DESCRIPTION
**Pull request by hhslepicka**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Attn. @klauer @ZLLentz

### CODE REVIEW

**Comment by klauer** [comment]
I agree to be a maintainer

File: recipes/happi/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  sha256: 6135928358a329e715758624cfcc95c902d4ddc936bc8e29dd2d356874611153
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
```
Comment (klauer) [inline recipes/happi/meta.yaml]:
```suggestion
    - python >=3.6
```

File: recipes/happi/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    - pip
+  run:
+    - python
```
Comment (klauer) [inline recipes/happi/meta.yaml]:
```suggestion
    - python >=3.6
```

**Comment by hhslepicka** [comment]
@conda-forge/help-python this is good to go

File: recipes/happi/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  license: SLAC Open
+  license_family: OTHER
+  license_file: LICENSE.md
+  summary: 'Heuristic Access to Positions of Photon Instruments'
+
+extra:
+  recipe-maintainers:
+    - hhslepicka
+    - klauer
+    - ZLLentz
```
Comment (scopatz) [inline recipes/happi/meta.yaml]: @ZLLentz are you OK with being a maintainer?

Comment (ZLLentz) [reply recipes/happi/meta.yaml]: I agree to be a maintainer

**Comment by npavlovikj** [comment]
Thank you for contributing @hhslepicka 🎉

**Pull request merged**